keps/sig-node/4033-group-driver-detection-over-cri/README.md

```diff
@@ -500,80 +263,41 @@ ignored by kubelet and it will resort to its own configuration settings. Note:
-Kubernetes are observable, scalable and supportable; can be safely operated in
-production environments, and can be disabled or rolled back in the event they
-cause increased failures in production. See more in the PRR KEP at
-https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
-
-The production readiness review questionnaire must be completed and approved
-for the KEP to move to `implementable` status and be included in the release.
-
-In some cases, the questions below should also have answers in `kep.yaml`. This
-is to enable automation to verify the presence of the review, and to reduce review
-burden and latency.
+In GA, the fallback behavior will be removed, and the kubelet will rely on the
+container runtime to implement the feature. In practice, this means the cluster
+must use at least containerd v2.0 or cri-o v1.28 as a prerequisite for
+upgrading.
 
-The KEP must have a approver from the
-[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
-team. Please reach out on the
-[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
-you need any help or guidance.
--->
+## Production Readiness Review Questionnaire
 
 ### Feature Enablement and Rollback
 
-<!--
-This section must be completed when targeting alpha to a release.
--->
-
 ###### How can this feature be enabled / disabled in a live cluster?
 
-<!--
-Pick one of these and delete the rest.
-
-Documentation is available on [feature gate lifecycle] and expectations, as
-well as the [existing list] of feature gates.
-
-[feature gate lifecycle]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
-[existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
--->
-
 - [X] Feature gate (also fill in values in `kep.yaml`)
   - Feature gate name: KubeletCgroupDriverFromCRI
   - Components depending on the feature gate: kubelet
 
 ###### Does enabling the feature change any default behavior?
 
-<!--
-Any change of default behavior may be surprising to users or break existing
-automations, so be extremely careful here.
--->
+Yes.
 
-Yes. If/when the runtime is updated to a version that supports this, kubelet
+In alpha and beta, when the runtime is updated to a version that supports this, kubelet
 will ignore the cgroupDriver config option/flag. However, this change in
 behavior should not cause any breakages (on the contrary, it should fix
 scenarios where the kubelet `--cgroup-driver` setting is incorrectly
 configured). With old versions of the container runtimes (that don't support
 the new field in the CRI API) the default behavior is not changed.
 
-###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
-
-<!--
-Describe the consequences on existing workloads (e.g., if this is a runtime
-feature, can it break the existing applications?).
+In GA, the fallback behavior (and the kubelet `--cgroup-driver` flag) is
```

Comment:
we require a few releases before removing the flag as per deprecation policy. Please update. (The flag can do nothing, it just should be around marked as deprecated)

Reply:
There are more than a few places within our own repos that are still setting this flag
https://cs.k8s.io/?q=cgroup(.)%3Fdriver&i=nope&literal=nope&files=&excludeFiles=&repos=

And then users may be opting to configure it indirectly, e.g. through kubeadm or kops configurations.

Reply:
Check, I updated the PR to align with the official deprecation policy.